You might want to copy/paste the following content to your bug report. ,-)

***

	### Issue

	### Used configuration options

	* maxsyncaccounts = 1
	* ui = basic
	* ignore-readonly = no
	* pythonfile = 
	* socktimeout = 60

	* [mbnames] (details)

	* account name: 
	* localrepository = 
	* remoterepository = 
